Updated Arabic Language

Motaz-Alnuweiri opened the pull request:
I updated the Arabic language translations :)

Name: Motaz Alnuweiri
Email: sniper_moz@hotmail.com
Default.isl Version: 5.5.3+

martijnlaan commented:
Thanks but if you have fixes, comments or suggestions about a particular translation, please contact its maintainer.

Motaz-Alnuweiri commented:
You just accept my pull requests and add my name as I updated the translations with the original name that make the translations :)

**Note:** I added my name and the name of original translations to TranslatorNote
